anvio/__init__.py

```diff
@@ -844,6 +844,31 @@ def get_args(parser):
                       filtering criteria. In other programs, you may get everything, nothing, or an error. It really depends\
                       on the situation. Fortunately, mistakes are cheap, so it's worth a try."}
                 ),
+    'flank-mode': (
+            ['-f','--flank-mode'],
+            {'action': 'store_true',
+             'help':
+             "If in --flank-mode, anvi-export-locus will extract a locus based on the coordinates \
+              of flanking genes. You MUST provide 2 flanking genes in the form of TWO \
+              --search-term, --gene-caller-ids, or --hmm-sources. The --flank-mode option is  \
+              appropriate for extracting loci of variable gene number lengths, but are consistantly \
+              located between the same flanking genes in the genome(s) of interest."}
+              ),
+    'num-genes': (
+            ['-n','--num-genes'],
+            {'metavar': 'NUM_GENES',
+             'type': str,
+             'help':
+             "Required for DEFAULT mode. For each match (to the function, or HMM that was searched) a sequence which includes \
```

Comment:
please consider not adding a new line and keep value for dicts start with the same line as their corresponding keys :) you can take a look at other instances to see how the rest of the file is organized.

Reply:
Thanks Meren, I went back and fixed this style inconsistency